### ISSUE TITLE
10. relationship to the issue of potential under/overfitting is not

### ISSUE THREAD

**Issue by NickleDave** [yardencsGitHub/tweetynet#78]
> 10. Although the choice of a specific activation function is a relatively minor point, its relationship to the issue of potential under/overfitting is not. Overfitting for some functions can be improved by adding dropout [8]. Throughout the network description and results, a much more detailed discussion of potential overfitting is important. Currently there is no mention(!). It makes sense that the complexity of the network should scale to the complexity of the song, but it's not clear if the authors have a method in mind to guide this choice and avoid overfitting

- [ ] explain that we use validation set + patience to prevent overfitting
- [ ] explain that we use a fixed (not changing) test set to measure all results, which demonstrates lack of overfitting
- [ ] do this in both methods and results

**Comment by NickleDave** [comment]
We say this in methods, need to make sure it's crystal clear in results also and that we point this out in response to reviewers

**Comment by NickleDave** [comment]
All check boxes above met, closing
